contrib/guix/manifest.scm

```diff
@@ -10,17 +10,31 @@
              (gnu packages file)
              (gnu packages gawk)
              (gnu packages gcc)
+             (gnu packages installers)
              (gnu packages linux)
+             (gnu packages mingw)
              (gnu packages perl)
              (gnu packages pkg-config)
              (gnu packages python)
              (gnu packages shells)
+             (guix build-system gnu)
              (guix build-system trivial)
              (guix gexp)
              (guix packages)
              (guix profiles)
              (guix utils))
 
+(define (make-ssp-fixed-gcc xgcc)
+  "Given a XGCC package, return a modified package that uses the SSP function
+from glibc instead of from libssp.so. Taken from:
+http://www.linuxfromscratch.org/hlfs/view/development/chapter05/gcc-pass1.html"
```

Comment:
Could mention that you can ensure this is working by checking that we aren't linked against `libssp.so`. Although symbol-check will be taking care of this as well.

Reply:
Fixed in e63cb74749aa3b82c04fc9dacd963d3de00784b9